LICENSE

```diff
@@ -1,21 +1,2 @@
-MIT License
```

Comment:
I think we want to keep the MIT license similar to what main package currently has.

Reply:
I think they are the same. I added the license using `usethis::use_mit_license()`.